var/spack/repos/builtin/packages/openpmd-api/package.py

```diff
@@ -172,17 +172,9 @@ def check(self):
             # later tests
             ctest("--output-on-failure", "-j1")
 
-    def test(self):
-        """Perform smoke tests on the installed package."""
-        exes = ["openpmd-ls"]  # in 0.11.1+
-        for exe in exes:
-            spec_vers_str = "{0}".format(self.spec.version)
-            reason = "test version of {0} is {1}".format(exe, spec_vers_str)
-            self.run_test(
-                exe,
-                ["--version"],
-                [spec_vers_str],
-                installed=True,
-                purpose=reason,
-                skip_missing=False,
-            )
+    def test_run_openpmd_ls(self):
+        """Test if openpmd-ls runs"""
+        if self.spec.satisfies("0.11.0"):
+            raise SkipTest("Package must be installed as version 0.11.1 or later")
+        exe = which(join_path(self.prefix.bin, "openpmd-ls"))
```

Comment:
The original version of the code checks the output (see original lines 179, 183-184.

Therefore this is a test that does check that the executable runs correctly.